src/test/java/com/google/javascript/gents/multiTests/provide_imports/export.ts

```diff
@@ -1,17 +1,15 @@
 export function B() {}
 
-export const x = 4;
-
-export const y = 8;
+export const C = {};
```

Comment:
this won't work because the object `C` in TypeScript never defines that it has a property `x`. We'd have to generate:

```
C = {
  x: ...,
};
```